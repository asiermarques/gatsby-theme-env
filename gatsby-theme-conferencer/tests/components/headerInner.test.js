import React from "react";
import { describe, it } from "@jest/globals";
import renderer from "react-test-renderer";
import Header from "../../src/components/HeaderInner";
import HeaderConfigFixture from "../__fixtures/HeaderConfig";

describe("HeaderInner", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header data={HeaderConfigFixture} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
