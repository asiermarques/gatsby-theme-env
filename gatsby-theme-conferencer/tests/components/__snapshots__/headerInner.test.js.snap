// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderInner renders correctly 1`] = `
<div
  className="jumbotron"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-7"
      >
        <a
          href="/"
        >
          <h2
            className="display-4"
          />
        </a>
        <p
          className="lead"
        />
      </div>
      <div
        className="col-sm-5 text-center"
      >
        <p />
        <a
          className="btn btn-primary btn-lg"
          role="button"
        />
      </div>
    </div>
  </div>
</div>
`;
